### PR TITLE
Migrate the CPU stress test to the chaosapi

### PR DESCRIPTION
**Pull request by bugraderre**
This commit migrates the stress_cpu script completely to the chaosapi.
This commit serves as a first discussion of the new feature. Let's
discuss in the PR whether changes are necessary and relevant. If
everything is okay the next scripts will be moved:

* Are the function docs appropriate and should there be any information
that states that a working internet connection is necessary to make use
of the scripts?
* Is the design and architecture okay

### CODE REVIEW

File: pdchaosazure/machine/actions.py [hunk 1]
```diff
@@ -161,19 +166,20 @@ def stress_cpu(filter: str = None,
...
     with concurrent.futures.ThreadPoolExecutor(max_workers=len(machines)) as executor:
         for machine in machines:
-            command_id, script_content = command.prepare(machine, 'cpu_stress_test')
+            command_id, script_content = command.prepare(machine, operation_name)
             parameters = {
                 'command_id': command_id,
                 'script': [script_content],
                 'parameters': [
-                    {'name': "duration", 'value': duration}
+                    {'name': "input_duration", 'value': duration},
```
Comment (mkaszub) [inline pdchaosazure/machine/actions.py:174]: any reason for prefixing it with "input_"?

File: pdchaosazure/machine/actions.py [hunk 2]
```diff
@@ -136,6 +136,7 @@ def restart_machines(filter: str = None,
 
 def stress_cpu(filter: str = None,
                duration: int = 120,
+               number_of_cores: int = 0,
```
Comment (mkaszub) [inline pdchaosazure/machine/actions.py:139]: Let's get rid of cores as we cannot address particular cores or number of cores that we want to stress

File: pdchaosazure/machine/actions.py (hunk 1 above)
Comment (bugraderre) [reply pdchaosazure/machine/actions.py:174]: Yes. Readability inside the script. From the perspective of the script it's more easy to see which values are *input*ed and which variables are calculated inside the script.

File: pdchaosazure/machine/actions.py (hunk 2 above)
Comment (bugraderre) [reply pdchaosazure/machine/actions.py]: Done

**Pull request merged**